modules/nw-scaling-ingress-controller.adoc

```diff
@@ -0,0 +1,93 @@
+// Module filename: nw-scaling-ingress-controller.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-ingress-controller-configuration-{context}"]
+= Scaling an ingress controller
+
+An ingress controller can be manually scaled up or down based on
+requirements such as the need to increase throughput. `oc`
+commands are used to scale the `IngressController` resource.
```

Comment:
When referring to the resource name like this, should we use camel casing? All the REST API and CLI tooling seem to refer to resource names in lowercase.

Reply:
IMO we should use the API object `kind` value to be consistent with upstream. For example, [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/).

@bmcelvee is there a docs standard we should follow?